site/styles/_carousel.scss

```diff
@@ -50,17 +48,24 @@ s72-carousel {
       }
     }
     img {
+      position: absolute;
       height: $carousel-height;
-      margin-left: 50%;
-      transform: translateX(-50%);
-
+      top: 50%;
+      left: 50%;
+      transform: translate(-50%, -50%);
       @include media-breakpoint-up(sm) {
         height: $carousel-height-sm;
       }
       @include media-breakpoint-up(md) {
         width: auto;
         height: $carousel-height-md;
       }
+      // Finds the width of the image given the carousel height and a fixed aspect ratio of 1600:600
+      $carousel-image-width: (1600 / 600) * $carousel-height-md;
```

Comment:
Does this work at the default and sm breakpoints too?

Reply:
Yes. `$carousel-image-width` is used to detect when the screen width is greater than the image width. This isn't going to happen with small screen widths.

Reply:
Is this doing the same as `imageSizeRatio` [in the component](https://github.com/indiereign/shift72-web/blob/master/src/ui/carousel/carousel.component.js#L10)?

Reply:
I think that ratio should be variable(s) so if a site has different carousel image size this code should still be reasonably happy.

I also think someone (me, @PeculiarGoat, anyone) should add an attribute to the carousel component to disable resizing which might make these changes a little easier to wrangle.

Reply:
@stajs No. That code is calculating the height of `.s72-carousel-slides` from the image width. When the image width changes, the height style changes in `.s72-carousel-slides`.  I don't think it's being used though as it is overwritten by the `carousel-height` variables.
![image](https://user-images.githubusercontent.com/41402426/104655675-ea84ac00-5722-11eb-8e62-fa7df9208085.png)

Reply:
@gscragg Agreed. I'll add it in.